wp-content/themes/exploore-40/demo-content/theme-options.txt
{"last_tab":"7","slz-layout":"1","slz-layout-boxed-bg":{"background-color":"#ffffff","background-repeat":"no-repeat","background-size":"cover","background-attachment":"","background-position":"center center","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"slz-site-skin":"color-1","slz-logo-header":{"url":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/themes\/exploore\/assets\/public\/images\/logo\/logo-2.png","id":"","height":"","width":"","thumbnail":""},"slz-logo-header-transparent":{"url":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/themes\/exploore\/assets\/public\/images\/logo\/logo.png","id":"","height":"","width":"","thumbnail":""},"slz-logo-footer":{"url":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/themes\/exploore\/assets\/public\/images\/logo\/logo.png","id":"","height":"","width":"","thumbnail":""},"slz-logo-footer-light":{"url":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/Logo-2.png","id":"566","height":"52","width":"206","thumbnail":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/Logo-2-150x52.png"},"slz-sticky-enable":"1","slz-backtotop":"1","slz-currency-sign":"$","slz-symbol-currency-position":"before","slz-search-general":{"disabled":{"placebo":"placebo"},"enabled":{"placebo":"placebo","tour":"Tour","hotel":"Hotel","car":"Car Rent","cruise":"Cruises"}},"slz-search-tour-icon":"flaticon-people","slz-search-hotel-icon":"flaticon-three","slz-search-car-icon":"flaticon-transport-7","slz-search-cruises-icon":"flaticon-transport-4","slz-social-facebook":"http:\/\/facebook.com","slz-social-twitter":"http:\/\/twitter.com","slz-social-google-plus":"https:\/\/plus.google.com\/","slz-social-pinterest":"https:\/\/pinterest.com\/","slz-social-instagram":"http:\/\/instagram.com","slz-social-dribbble":"http:\/\/dribbble.com","slz-social-share":{"disabled":{"placebo":"placebo","pinterest":"Pinterest","linkedin":"LinkedIn","digg":"Digg"},"enabled":{"placebo":"placebo","facebook":"Facebook","twitter":"Twitter","google-plus":"Google Plus"}},"slz-style-header":"five","slz-logo-header-04":{"url":"","id":"","height":"","width":"","thumbnail":""},"slz-header-search-icon":"1","slz-header-search-type":"2","slz-header-dropdonw-language":"0","slz-header-dropdonw-usd":"0","slz-header-social-info":"1","slz-header-social":{"disabled":{"placebo":"placebo","instagram":"Instagram","dribbble":"Dribbble"},"enabled":{"placebo":"placebo","facebook":"Facebook","google-plus":"Google plus","twitter":"Twitter","pinterest":"Pinterest"}},"slz-header-other-info":[""],"slz-header-account":"theme","slz-menu-custom":"","slz-menu-item-text":{"regular":"#555e69","hover":"#ffdd00","active":"#ffdd00"},"slz-menu-height":{"height":"100px","units":"px"},"slz-submenu-custom":"","slz-submenu-bg":{"color":"#fff","alpha":"1","rgba":"rgba(255, 255, 255, 1)"},"slz-submenu-color":{"regular":"#555e69","hover":"#555e69","active":"#555e69"},"slz-submenu-active-color":{"color":"#f5f5f5","alpha":"1","rgba":"rgba(245,245,245,1)"},"slz-submenu-width":{"width":"px","units":"px"},"slz-submenu-border":{"border-top":"0px","border-right":"0px","border-bottom":"2px","border-left":"0px","border-style":"solid","border-color":"#ffdd00"},"slz-submenu-padding":{"padding-top":"","padding-right":"","padding-bottom":"","padding-left":""},"slz-dropdownmenu-align":"right","slz-page-title-show":"1","slz-page-title-bg":{"background-color":"#f3f3f3","background-repeat":"no-repeat","background-size":"cover","background-attachment":"fixed","background-position":"center center","background-image":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/blog.jpg","media":{"id":"502","height":"1000","width":"1920","thumbnail":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/blog-150x150.jpg"}},"slz-pagetitle-overlay-bg":{"color":"#34495e","alpha":"0.23","rgba":"rgba(52,73,94,0.23)"},"slz-page-title-height":{"height":"540px","units":"px"},"slz-pagetitle-align":"left","slz-show-title":"1","slz-pagetitle-title":{"font-weight":"900","font-style":"","text-transform":"uppercase","font-size":"100px","color":"#ffffff"},"slz-show-breadcrumb":"1","slz-breadcrumb-path":{"font-weight":"400","font-style":"","text-transform":"uppercase","font-size":"20px","color":"#ffffff"},"slz-breadcrumb-path2":{"font-weight":"400","font-style":"","text-transform":"uppercase","font-size":"12px"},"slz-breadcrumb-border-color":"#ffdd00","slz-sidebar-layout":"none","slz-sidebar":"","slz-blog-sidebar-layout":"right","slz-blog-sidebar":"slzexploore-sidebar-blog","slz-tour-sidebar-layout":"none","slz-tour-sidebar":"","slz-hotel-sidebar-layout":"none","slz-hotel-sidebar":"","slz-car-sidebar-layout":"left","slz-car-sidebar":"slzexploore-sidebar-main","slz-cruises-sidebar-layout":"left","slz-cruises-sidebar":"slzexploore-sidebar-shop","slz-shop-sidebar-layout":"none","slz-shop-sidebar":"","slz-sidebar-mb":{"margin-bottom":"50px"},"slz-sidebar-pb":{"padding-bottom":""},"slz-subcribe":"1","slz-subcribe-text-1":"Subscribe <span class=\"logo-text\">Exploore<\/span>\u00a0to get latest offers and deals to day","slz-subcribe-text-2":"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore.","slz-subcribe-form-text":"Write your email","slz-footer":"1","slz-footerbt-logo-show":"1","slz-footer-col":"5","slz-footer-style":"dark","slz-footer-bg":{"background-color":"#292F32","background-repeat":"no-repeat","background-size":"cover","background-attachment":"fixed","background-position":"center center","background-image":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/bg-footer.jpg","media":{"id":"256","height":"1000","width":"1920","thumbnail":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/bg-footer-150x150.jpg"}},"slz-footer-mask-bg":{"color":"transparent","alpha":"0","rgba":"rgba(0, 0, 0, 0)"},"slz-footer-light-bg":{"background-color":"#ffffff","background-repeat":"no-repeat","background-size":"cover","background-attachment":"","background-position":"center center","background-image":"","media":{"id":"","height":"","width":"","thumbnail":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/bg-footer-2-150x150.png"}},"slz-footer-light-mask-bg":{"color":"transparent","alpha":"0","rgba":"rgba(0, 0, 0, 0)"},"slz-footer-light-padding":{"padding-top":"0","padding-bottom":"0"},"slz-footerbt-show":"1","slz-footerbt-partner":"1","slz-footerbt-partner-cat":["91"],"slz-footerbt-social-content":"1","slz-footer-social":{"disabled":{"placebo":"placebo"},"enabled":{"placebo":"placebo","facebook":"Facebook","google-plus":"Google Plus","twitter":"Twitter","instagram":"Instagram","dribbble":"Dribbble","pinterest":"Pinterest"}},"slz-footerbt-text":"\u00a9 Designed by SWLABS.","slz-blog-show-title":"category","slz-bloginfo":{"disabled":{"placebo":"placebo"},"enabled":{"placebo":"placebo","author":"Author","view":"View","comment":"Comment","date":"Date"}},"slz-blog-tag":"1","slz-blog-cat":"1","slz-commentbox":"1","slz-tour-show-title":"1","slz-tour-custom-title":"Tours","slz-tour-detail-custom-title":"Tour","slz-tour-show-price":"1","slz-tour-show-related":"1","slz-tour-posts":"4","slz-tour-page-title-show":"1","slz-tour-page-title-bg":{"background-color":"#f3f3f3","background-repeat":"no-repeat","background-size":"cover","background-attachment":"fixed","background-position":"center center","background-image":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/tour-result.jpg","media":{"id":"418","height":"1000","width":"1920","thumbnail":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/tour-result-150x150.jpg"}},"slz-tour-top-cat":"1","slz-tour-filter":{"disabled":{"placebo":"placebo","category":"Categories"},"enabled":{"placebo":"placebo","find-tour-box":"Find Tour Box","price":"Price","rating":"Review Rating","cities":"Locations"}},"slz-tour-price-max":"10000","slz-tour-result-count":"1","slz-tour-list-sort":"1","slz-tour-list-info":{"disabled":{"placebo":"placebo"},"enabled":{"placebo":"placebo","comment":"Review","view":"View","list-wist":"Wishlist"}},"slz-hotel-show-title":"1","slz-hotel-custom-title":"Hotels","slz-hotel-detail-custom-title":"","slz-hotel-show-price":"1","slz-hotel-show-map":"1","slz-hotel-show-related":"1","slz-hotel-posts":"5","slz-hotel-page-title-show":"1","slz-hotel-page-title-bg":{"background-color":"#f3f3f3","background-repeat":"no-repeat","background-size":"cover","background-attachment":"fixed","background-position":"center center","background-image":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/hotel-result.jpg","media":{"id":"366","height":"1000","width":"1920","thumbnail":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/hotel-result-150x150.jpg"}},"slz-hotel-filter":{"disabled":{"placebo":"placebo","review_rating":"Review Rating","location":"Locations"},"enabled":{"placebo":"placebo","find-hotel-box":"Find Hotel Box","accommodation_type":"Category","facilities":"Facilities","price":"Price","rating":"Star Rating"}},"slz-hotel-price-max":"10000","slz-hotel-max-rate":"5","slz-hotel-result-count":"1","slz-hotel-list-sort":"1","slz-hotel-list-info":{"disabled":{"placebo":"placebo"},"enabled":{"placebo":"placebo","comment":"Review","view":"View","list-wist":"Wishlist","share":"Share","address":"Address"}},"slz-car-show-title":"1","slz-car-custom-title":"CAR RENT","slz-car-detail-custom-title":"CAR RENT","slz-car-show-price":"1","slz-car-show-related":"1","slz-car-posts":"6","slz-car-page-title-show":"1","slz-car-page-title-bg":{"background-color":"#f3f3f3","background-repeat":"no-repeat","background-size":"cover","background-attachment":"fixed","background-position":"center center","background-image":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/about-us.jpg","media":{"id":"327","height":"1000","width":"1920","thumbnail":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/about-us-150x150.jpg"}},"slz-car-filter":{"disabled":{"placebo":"placebo"},"enabled":{"placebo":"placebo","find-car-box":"Find Cars Box","price":"Price","rating":"Review Rating","location":"Locations","category":"Categories"}},"slz-car-price-max":"10000","slz-car-result-count":"1","slz-car-list-sort":"1","slz-cruises-show-title":"1","slz-cruises-custom-title":"Cruises","slz-cruises-detail-custom-title":"Cruises","slz-cruises-show-price":"1","slz-cruises-show-related":"1","slz-cruises-posts":"4","slz-cruises-page-title-show":"1","slz-cruises-page-title-bg":{"background-color":"#f3f3f3","background-repeat":"no-repeat","background-size":"cover","background-attachment":"fixed","background-position":"center center","background-image":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/tour-result.jpg","media":{"id":"418","height":"1000","width":"1920","thumbnail":"http:\/\/wp.swlabs.co\/explooredev2\/wp-content\/uploads\/2016\/05\/tour-result-150x150.jpg"}},"slz-cruises-filter":{"disabled":{"placebo":"placebo","facility":"Facilities","star_rating":"Star Rating"},"enabled":{"placebo":"placebo","find-cruises-box":"Find Cruises Box","price":"Price","rating":"Review Rating","location":"Locations","category":"Categories"}},"slz-cruises-price-max":"10000","slz-cruises-max-rate":"5","slz-cruises-result-count":"1","slz-cruises-list-sort":"1","slz-booking-active":{"disabled":{"placebo":"placebo"},"enabled":{"placebo":"placebo","car":"Car Rent","cruise":"Cruises","hotel":"Hotel","tour":"Tour"}},"slz-tour-confirm-email-to":"","slz-tour-confirm-email-from":"info@swlabs.co","slz-tour-confirm-email-subject":"Exploore \"[tour_name]\"","slz-tour-confirm-email-header":"Reply-To: [site_name] <[from_email]>","slz-tour-confirm-email-description":"<table style=\"width: 580px; margin: 0; padding: 0; background: #ffffff; font-family: arial;\" border=\"0\" cellspacing=\"0\" cellpadding=\"0\">\r\n<tbody>\r\n<tr>\r\n<td style=\"padding-top: 20px; padding-left: 10px; padding-right: 10px;\" valign=\"top\"><a title=\"[site_name]\" href=\"[home_url]\" target=\"_blank\">\r\n<img class=\"CToWUd\" style=\"outline: none; text-decoration: none; border: none;\" src=\"[logo_url]\" alt=\"[site_name]\" width=\"160\" \/>\r\n<\/a><\/td>\r\n<\/tr>\r\n<tr>\r\n<td style=\"padding: 20px 10px 20px 10px;\" valign=\"top\">\r\n<div style=\"font-size: 18px; line-height: 22px; font-family: arial; color: #003580; font-weight: bold;\">Thanks, [customer_first_name] [customer_last_name]! Your reservation is now confirmed.<\/div><\/td>\r\n<\/tr>\r\n<tr>\r\n<td style=\"padding: 20px 10px 20px 10px;\" valign=\"top\">\r\n<div style=\"text-align: left; font-size: 16px; line-height: 21px; font-family: arial; color: #333; vertical-align: middle;\"><a title=\"Tour info\" href=\"[tour_url]\" target=\"_blank\"><b>[tour_name]<\/b><\/a><\/div><\/td>\r\n<\/tr>\r\n<tr>\r\n<td style=\"padding: 20px 10px 20px 10px;\" valign=\"top\">\r\n<div style=\"text-align: left; font-size: 16px; line-height: 21px; font-family: arial; color: #333; vertical-align: middle;\">[tour_description]<\/div><\/td>\r\n<\/tr>\r\n<tr>\r\n<td style=\"padding-top: 5px; text-align: left;\" valign=\"top\">\r\n<table style=\"margin: 0px; padding: 0px; border: 0px;\" border=\"0\" cellspacing=\"0\" cellpadding=\"0\">\r\n<tbody>\r\n<tr>\r\n<td style=\"text-align: left;\" valign=\"top\" width=\"100\">[tour_thumbnail]<\/td>\r\n<td style=\"text-align: left; font-family: arial; color: #333; font-size: 12px; line-height: 17px; padding-left: 20px;\" valign=\"top\">\r\n<table style=\"margin: 0px; padding: 0px; border: 0px;\" border=\"0\" width=\"100%\" cellspacing=\"0\" cellpadding=\"0\">\r\n<tbody>\r\n<tr>\r\n<td style=\"text-align: left; font-family: arial; color: #333; font-size: 12px; line-height: 17px; padding-right: 10px;\" valign=\"top\"><b>Start Date:<\/b><\/td>\r\n<td style=\"text-align: left; font-family: arial; color: #333; font-size: 12px; line-height: 17px;\" valign=\"top\">[tour_date]<\/td>\r\n<\/tr>\r\n<tr>\r\n<td style=\"text-align: left; font-family: arial; color: #333; font-size: 12px; line-height: 17px; padding-right: 10px;\" valign=\"top\"><b>Duration:<\/b><\/td>\r\n<td style=\"text-align: left; font-family: arial; color: #333; font-size: 12px; line-height: 17px;\" valign=\"top\">[tour_duration]<\/td>\r\n<\/tr>\r\n<tr>\r\n<td style=\"text-align: left; font-family: arial; color: #333; font-size: 12px; line-height: 17px; padding-right: 10px;\" valign=\"top\"><b>Destination:<\/b><\/td>\r\n<td style=\"text-align: left; font-family: arial; color: #333; font-size: 12px; line-height: 17px;\" valign=\"top\">[tour_destination]<\/td>\r\n<\/tr>\r\n<\/tbody>\r\n<\/table>\r\n<\/td>\r\n<\/tr>\r\n<\/tbody>\r\n<\/table>\r\n<\/td>\r\n<\/tr>\r\n<\/tbody>\r\n<\/table>\r\n<table style=\"margin: 0px; padding: 0px; border: 0px;\" border=\"0\" width=\"100%\" cellspacing=\"0\" cellpadding=\"0\">\r\n<tbody>\r\n<tr>\r\n<td style=\"text-align: left; font-family: arial; color: #333; font-size

wp-content/themes/exploore-40/exploore/framework/config.php
<?php
return array(
	'load' => array(
		// Register widgets
		array( 'add_action', 'widgets_init', array( SLZEXPLOORE_THEME_CLASS, '[widget.Widget_Init, load]') ),
		// action inline css
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_add_inline_style', array( SLZEXPLOORE_THEME_CLASS, '[theme.Theme_Init, add_inline_style]') ),
		// get page options
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_page_options',      array( SLZEXPLOORE_THEME_CLASS, '[theme.Theme_Init, get_page_options]') ),
		// show index content
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_show_index',        array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_post_index]') ),
		// Frontend actions

		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_show_header',       array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, header]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_show_footerbottom', array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, footerbottom]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_show_breadcrumb',   array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, breadcrumb]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_entry_thumbnail',   array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_post_entry_thumbnail]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_entry_video',       array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_post_entry_video]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_entry_meta',        array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_post_entry_meta]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_tags_meta',         array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_post_tags_meta]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_categories_meta',         array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_post_categories_meta]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_show_page_title',   array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_page_title]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_show_slider',       array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_slider]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_show_searchform',   array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_searchform]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_post_author',       array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_post_author]') ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_show_posttype_header_content',       array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_posttype_header_content]') ),

		// share post
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_share_link',        array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, get_share_link]') ),
		// share hotel, tour, car and cruise
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_share_custom_post', array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, share_custom_post]') ),

		// login
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_login_link',        array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, show_login_link]') ),

		array( 'add_action', 'comment_post',                 array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, add_comment_rating]') ),
		array( 'add_action', 'deleted_comment',              array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, update_review_rating]' ) ),
		array( 'add_action', 'trashed_comment',              array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, update_review_rating]' ) ),
		array( 'add_action', 'untrashed_comment',            array( SLZEXPLOORE_THEME_CLASS, '[front.Top_Controller, update_review_rating]' ) ),
		// User Controller
		array( 'add_action', 'after_setup_theme',                             array( SLZEXPLOORE_THEME_CLASS, '[profile.User_Controller, create_default_pages]') ),
		array( 'add_action', 'wp_loaded',                                     array( SLZEXPLOORE_THEME_CLASS, '[profile.User_Controller, process_login]'), 20 ),
		array( 'add_action', 'wp_loaded',                                     array( SLZEXPLOORE_THEME_CLASS, '[profile.User_Controller, process_registration]'), 20 ),
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_print_notices',     array( SLZEXPLOORE_THEME_CLASS, '[profile.User_Controller, print_notices]') ),

	),
	'init' => array(
		// Regist Menu
		array( 'register_nav_menu', 'main-nav',   esc_html__( 'Main Navigation', 'exploore' ) ),
		array( 'register_nav_menu', 'page-404-nav', esc_html__( '404 Pages', 'exploore' ) ),

		// Ajax
		array( 'add_action', 'wp_ajax_slzexploore',        array( SLZEXPLOORE_THEME_CLASS, '[Application, ajax]' ) ),
		array( 'add_action', 'wp_ajax_nopriv_slzexploore', array( SLZEXPLOORE_THEME_CLASS, '[Application, ajax]' ) ),

		// Welcome page
		array( 'add_action', 'admin_menu', array( SLZEXPLOORE_THEME_CLASS, '[theme.Theme_Controller, add_welcome]' ) ),
		array( 'add_action', 'admin_init', array( SLZEXPLOORE_THEME_CLASS, '[theme.Theme_Controller, call_tgm_plugin_action]' ) ),

		// Add sidebar area
		array( 'add_action', 'admin_print_scripts',                    array( SLZEXPLOORE_THEME_CLASS, '[theme.Widget_Init, add_widget_field]' ) ),
		array( 'add_action', 'load-widgets.php',                       array( SLZEXPLOORE_THEME_CLASS, '[widget.Widget_Init, add_sidebar_area]' ) ),
		array( 'add_action', 'wp_ajax_slzexploore_del_custom_sidebar', array( SLZEXPLOORE_THEME_CLASS, '[widget.Widget_Init, delete_custom_sidebar]' ) ),
	),
	'front_init' => array(),

	'admin_init' => array(
		// add action
		array( 'add_action', 'save_post',             array( SLZEXPLOORE_THEME_CLASS, '[Application, save]' ) ),
		array( 'add_action', 'admin_enqueue_scripts', array( SLZEXPLOORE_THEME_CLASS, '[theme.Theme_Init, admin_enqueue]' ) ),

		// init page options
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_init_page_setting',  array( SLZEXPLOORE_THEME_CLASS, '[theme.Theme_Init, init_page_setting]' ) ),
		array( 'do_action',  SLZEXPLOORE_THEME_PREFIX . '_init_page_setting'),

		// save_page
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_save_page',          array( SLZEXPLOORE_THEME_CLASS, '[theme.Theme_Init, save_page]') ),

		// add mbox page options
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_metabox_pageoption',       array( SLZEXPLOORE_THEME_CLASS, '[theme.Theme_Init, add_page_options]' ) ),
		array( 'do_action',  SLZEXPLOORE_THEME_PREFIX . '_metabox_pageoption' ),
		
		array( 'add_action', SLZEXPLOORE_THEME_PREFIX . '_get_theme_header',          array( SLZEXPLOORE_THEME_CLASS, '[theme.Theme_Controller, get_theme_header]') ),

	),

	'save_post' => array(
		'page'             => array( 'theme.Theme_Init', 'save_page' ),
		'post'             => array( 'theme.Theme_Init', 'save_post' ),
		'product'          => array( 'theme.Theme_Init', 'save_product' ),
	),
	'page_options' => array(
		'post_types' => array( 'post', 'page', 'slzexploore_hotel', 'slzexploore_tour', 'slzexploore_team', 'slzexploore_car', 'slzexploore_cruise', 'product' ),
	),
	'mapping' => array(
		'special_options' => array(
			'header_layout', 'header_top_show', 'header_logo_show', 'header_sticky_enable',
			'footer_show', 'footer_top_show', 'footer_bottom_show',
			'page_title_show', 'breadcrumb_show', 'title_show', 'sub_title_show'
		),
		'no-default-options' => array( 'no_default' ),
		'options' => array(
			'header' => array(
				'header_layout'                => 'slz-style-header',
				'header_sticky_enable'         => 'slz-sticky-enable',
				'header_top_show_menu'         => 'slz-header-top-show-menu',
				'header_top_menu_position'     => 'slz-header-top-menu-position',
				'header_top_menu'              => 'slz-header-top-menu'
			),
			'general' => array(
				'background_transparent'   => array( 'slz-layout-boxed-bg', 'background-color' ),
				'background_color'         => array( 'slz-layout-boxed-bg', 'background-color' ),
				'background_repeat'        => array( 'slz-layout-boxed-bg', 'background-repeat' ),
				'background_attachment'    => array( 'slz-layout-boxed-bg', 'background-attachment' ),
				'background_position'      => array( 'slz-layout-boxed-bg', 'background-position' ),
				'background_size'          => array( 'slz-layout-boxed-bg', 'background-size' ),
				'background_image'         => array( 'slz-layout-boxed-bg', 'background-image' ),
				'background_image_id'      => array( 'slz-layout-boxed-bg', 'media', 'id' ),
			),
			'page_title' => array(
				'page_title_show'          => 'slz-page-title-show',
				'breadcrumb_show'          => 'slz-show-breadcrumb',
				'title_show'               => 'slz-show-title',
				'title_custom_content'     => '',
				'pt_background_color'      => array( 'slz-page-title-bg', 'background-color' ),
				'pt_background_repeat'     => array( 'slz-page-title-bg', 'background-repeat' ),
				'pt_background_attachment' => array( 'slz-page-title-bg', 'background-attachment' ),
				'pt_background_position'   => array( 'slz-page-title-bg', 'background-position' ),
				'pt_background_size'       => array( 'slz-page-title-bg', 'background-size' ),
				'pt_background_image'      => array( 'slz-page-title-bg', 'background-image' ),
				'pt_background_image_id'   => array( 'slz-page-title-bg', 'media', 'id' ),
				'pt_height'                => array( 'slz-page-title-height', 'height' ),
			),
			'footer' => array(
				'footer_show'              => 'slz-footer',
				'show_newsletter'          => 'slz-subcribe',
				'footer_bottom_show'       => 'slz-footerbt-show',
				'footer_style'             => 'slz-footer-style',
			),
			'sidebar' => array(
				'sidebar_layout'           => 'slz-sidebar-layout',
				'sidebar_id'               => 'slz-sidebar',
				'sidebar_post_layout'      => 'slz-blog-sidebar-layout',
				'sidebar_post_id'          => 'slz-blog-sidebar',
				'sidebar_tour_layout'      => 'slz-tour-sidebar-layout',
				'sidebar_tour_id'          => 'slz-tour-sidebar',
				'sidebar_hotel_layout'     => 'slz-hotel-sidebar-layout',
				'sidebar_hotel_id'         => 'slz-hotel-sidebar',
				'sidebar_shop_layout'      => 'slz-shop-sidebar-layout',
				'sidebar_shop_id'          => 'slz-shop-sidebar',
				'sidebar_car_layout'       => 'slz-car-sidebar-layout',
				'sidebar_car_id'           => 'slz-car-sidebar',
				'sidebar_cruise_layout'    => 'slz-cruises-sidebar-layout',
				'sidebar_cruise_id'        => 'slz-cruises-sidebar',
			),
			'no_default' => array(
				'body_extra_class'         => 'slz-body-extra-class',
				'ct_padding_top'           => 'slz-content-padding-top',
				'ct_padding_bottom'        => 'slz-content-padding-bottom',
				'show_page_contact'        => 'slz-contact-section-1',
				'slider-header-fixed'      => 'slz-header-hide',
				'header_content_type'      => 'slz-header-content-type',
				'header_revolution_slider' => 'slz-header-slider',
				'header_search_form'       => 'slz-header-search-form',
				'header_bg_image'          => 'slz-header-bg-image',
				'header_bg_video'          => 'slz-header-bg-video',
				'header_caption_1'         => 'slz-header-caption-1',
				'font_size_1'              => 'slz-header-caption-fontsize-1',
				'header_caption_2'         => 'slz-header-caption-2',
				'font_size_2'              => 'slz-header-caption-fontsize-2',
				'header_button'            => 'slz-header-button',
				'header_button_hover'      => 'slz-header-button-hover',
				'button_link'              => 'slz-header-button-link',
			),
		),
		'archive' => array(
			'slzexploore_tour' => array(
				'slz-tour-page-title-show'    => 'slz-page-title-show',
				'slz-tour-page-title-bg'      => 'slz-page-title-bg',
			),
			'slzexploore_hotel' => array(
				'slz-hotel-page-title-show'    => 'slz-page-title-show',
				'slz-hotel-page-title-bg'      => 'slz-page-title-bg',
			),
			'slzexploore_car' => array(
				'slz-car-page-title-show'    => 'slz-page-title-show',
				'slz-car-page-title-bg'      => 'slz-page-title-bg',
			),
			'slzexploore_cruise' => array(
				'slz-cruises-page-title-show'    => 'slz-page-title-show',
				'slz-cruises-page-title-bg'      => 'slz-page-title-bg',
			),
		),
		'product' => array(
			'slz-shop-page-title-show'    => 'slz-page-title-show',
			'slz-shop-page-title-bg'      => 'slz-page-title-bg',
		),
	),
	'theme_options' => array(
		// default theme options
		'slz-style-header'         =>'two',
		'slz-dropdownmenu-align'   =>'right',
		'slz-page-title-show'      => '1',
		'slz-show-title'           => '1',
		'slz-show-breadcrumb'      => '1',
		'slz-footer'               => '1',
		'slz-footerbt-show'        => '1',
		'slz-footerbt-text'        => esc_html__('&copy; 2016 BY SWLABS. ALL RIGHT RESERVE.', 'exploore'),
		'slz-sidebar-layout'       => 'left',
		'slz-blog-sidebar-layout'  => 'right',
		'slz-404-title'             => esc_html__('404', 'exploore'),
		'slz-404-desc'             => esc_html__('Page not found.', 'exploore'),
		'slz-404-backhome'         => esc_html__('Go Back', 'exploore'),
		'slz-404-button-02'        => esc_html__('Get Help', 'exploore'),
		'slz-blog-show-title'      => 'category',
		'slz-header-account'       => 'hide',
	),
	'image_sizes' => array(
		// SLZEXPLOORE_THEME_PREFIX + name
		'-thumb-1140x515'        => array( 'width' => 1140, 'height' => 515 ),
		'-thumb-900x500'         => array( 'width' => 900, 'height' => 500 ),
		'-thumb-800x540'         => array( 'width' => 800, 'height' => 540 ),
		'-thumb-750x350'         => array( 'width' => 750, 'height' => 350 ),
		'-thumb-670x440'         => array( 'width' => 670, 'height' => 440 ),
		'-thumb-600x270'         => array( 'width' => 600, 'height' => 270 ),
		'-thumb-560x420'         => array( 'width' => 560, 'height' => 420 ),
		'-thumb-400x270'         => array( 'width' => 400, 'height' => 270 ),
		'-thumb-400x200'         => array( 'width' => 400, 'height' => 200 ),
		'-thumb-342x257'         => array( 'width' => 342, 'height' => 257 ),
		'-thumb-260x365'         => array( 'width' => 260, 'height' => 365 ),
		'-thumb-200x200'         => array( 'width' => 200, 'height' => 200 ),
		'-thumb-100x100'         => array( 'width' => 100, 'height' => 100 ),
		'-thumb-100x80'          => array( 'width' => 100, 'height' => 80 ),
		'-thumb-116x63'          => array( 'width' => 116, 'height' => 63 ),
		
		
	),
);